rviz_visual_tests/package.xml

```diff
@@ -0,0 +1,31 @@
+<package format="2">
+  <name>rviz_visual_tests</name>
+  <version>0.0.1</version>
+  <description>
+    3D testing framework for RViz.
+  </description>
+  <maintainer email="empty@empty.com">empty</maintainer>
```

Comment:
This needs to be set. If you guys cannot, please put me there `<maintainer email="william@openrobotics.org">William Woodall</maintainer>`